web/src/app/administration/components/workbasket-access-items/workbasket-access-items.component.ts

```diff
@@ -195,7 +198,20 @@ export class WorkbasketAccessItemsComponent implements OnInit, OnChanges, OnDest
     this.initialized = true;
   }
 
-  setAccessItemsGroups(accessItems: Array<WorkbasketAccessItems>) {
+  sortAccessItems(accessItems: WorkbasketAccessItems[], sortBy: string): WorkbasketAccessItems[] {
+    return accessItems.sort((a, b) => {
+      if (a[sortBy] < b[sortBy]) {
```

Comment:
what do you think if we change this to ternary statement 
`return a[sortBy] == b[sortBy] ? 0 : a[sortBy] > b[sortBy] ? 1 : -1`

I'm fine with either way

Reply:
I like those short expressions but in this case, I think the long version might be more readable..